jenkins_pipelines/environments/common/pipeline-pull-request-light.groovy

```diff
@@ -0,0 +1,208 @@
+                        }
+
+                        // Clean up old results
+                        sh "if [ -d ${resultdir} ];then ./clean-old-results -r ${resultdir};fi"
+
+                        // Create a directory for  to place the directory with the build results (if it does not exist)
+                        sh "mkdir -p ${resultdir}"
+
+                        // Clone sumaform
+                        sh "set +x; source /home/jenkins/.credentials set -x; ./terracumber-cli ${common_params} --gitrepo ${sumaform_gitrepo} --gitref ${sumaform_ref} --runstep gitsync"
+                    }
+                }
+            }
+            stage('Deploy') {
+                ws(environment_workspace){
+                    if(must_test) {
+                        // Passing the built repository by parameter using a environment variable to terraform file
+                        // TODO: We will need to add a logic to replace the host, when we use IBS for spacewalk
+                        env.PULL_REQUEST_REPO= "http://download.suse.de/ibs/SUSE:/Maintenance:/29643/SUSE_Updates_SLE-Product-SLES_15-SP1-LTSS_x86_64/"
+                        env.MASTER_REPO = "http://download.suse.de/ibs/SUSE:/Maintenance:/29643/SUSE_Updates_SLE-Product-SLES_15-SP1-LTSS_x86_64/"
+                        env.MASTER_OTHER_REPO = "http://download.suse.de/ibs/SUSE:/Maintenance:/29643/SUSE_Updates_SLE-Product-SLES_15-SP1-LTSS_x86_64/"
+                        env.MASTER_SUMAFORM_TOOLS_REPO = "http://download.suse.de/ibs/SUSE:/Maintenance:/29643/SUSE_Updates_SLE-Product-SLES_15-SP1-LTSS_x86_64/"
+                        env.TEST_PACKAGES_REPO = "http://download.suse.de/ibs/SUSE:/Maintenance:/29643/SUSE_Updates_SLE-Product-SLES_15-SP1-LTSS_x86_64/"
+                        env.UPDATE_REPO = "http://minima-mirror.mgr.prv.suse.net/jordi/some-updates/"
+                        if (additional_repo_url == '') {
+                            echo "Adding dummy repo for update repo"
+                            env.ADDITIONAL_REPO_URL = "http://minima-mirror.mgr.prv.suse.net/jordi/dummy/"
+                        } else {
+                            echo "Adding ${additional_repo_url}"
+                            env.ADDITIONAL_REPO_URL = additional_repo_url
+                        }
+                        env.SLE_CLIENT_REPO = "http://download.suse.de/ibs/SUSE:/Maintenance:/29643/SUSE_Updates_SLE-Product-SLES_15-SP1-LTSS_x86_64/"
+                        env.RHLIKE_CLIENT_REPO = "http://download.suse.de/ibs/SUSE:/Maintenance:/29643/SUSE_Updates_SLE-Product-SLES_15-SP1-LTSS_x86_64/"
+                        env.DEBLIKE_CLIENT_REPO = "http://download.suse.de/ibs/SUSE:/Maintenance:/29643/SUSE_Updates_SLE-Product-SLES_15-SP1-LTSS_x86_64/"
+                        env.OPENSUSE_CLIENT_REPO = "http://download.suse.de/ibs/SUSE:/Maintenance:/29643/SUSE_Updates_SLE-Product-SLES_15-SP1-LTSS_x86_64/"
```

Comment:
These URLS pointing MI repos will need to be parametrized. As MI repos are removed as soon as released.